app/domain/models/account.py

```
@@ -12,26 +13,24 @@ class Currency(StrEnum):
     avg_buy_price: Decimal
     unit: Currency
 
-    @property
+    @computed_field
     def total_value(self) -> Decimal:
         return self.balance * self.avg_buy_price
 
 
-@dataclass
-class Account:
+class Account(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)